app/modules/medication/index.js

```
@@ -0,0 +1,789 @@
+          return;
+        }
+        return withBusy(target, async () => {
+          await adjustStock(id, Math.trunc(delta), {
+            reason: 'card:restock',
+            dayIso: currentDayIso
+          });
+          updateFormStatus(
+            `Bestand ${delta > 0 ? '+' : ''}${Math.trunc(delta)} gespeichert.`
+          );
+        });
+        return;
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)